xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -544,6 +548,71 @@ rebuild_master:
         project: home:Admin
         package: ctris</screen>
         </sect4>
+
+        <sect4 xml:id="sec.obs.obs_scm_ci_workflow_integration.obs_workflows.steps.set_flags">
+          <title>Set Flags for Projects, Packages, Repositories or Architectures</title>
+
+          <para>Providing the flag <emphasis>build</emphasis>, the status <emphasis>enable</emphasis> and the project <emphasis>home:Admin</emphasis> will enable <emphasis>all</emphasis> builds:</para>
+
+          <itemizedlist>
+            <listitem>
+              <para>for the <emphasis>home:Admin:$MY_SCM_ORG:$MY_SCM_PROJECT:PR-$MY_PR_NUMBER</emphasis> project
+              when the webhook event is a pull request.</para>
+            </listitem>
+            <listitem>
+              <para>for the <emphasis>home:Admin</emphasis> project when the webhook event is a push.</para>
+            </listitem>
+          </itemizedlist>
+
+          <para>Please note that the provided project has to be a project which was a target project from a previous step like <emphasis>link_package</emphasis>
+          or <emphasis>branch_package</emphasis>. Same for the package, it has to be branched/linked in a previous step.</para>
+
+          <para>The type has to be one of following values:</para>
+          <itemizedlist>
+            <listitem><para>lock</para></listitem>
+            <listitem><para>build</para></listitem>
+            <listitem><para>publish</para></listitem>
+            <listitem><para>debuginfo</para></listitem>
+            <listitem><para>useforbuild</para></listitem>
+            <listitem><para>binarydownload</para></listitem>
+            <listitem><para>sourceaccess</para></listitem>
+            <listitem><para>access</para></listitem>
+          </itemizedlist>
+
+          <para>The status is either <emphasis>disable</emphasis> or <emphasis>enable</emphasis>.</para>
+
+          <para>Providing multiple flags is supported as noted in the configuration file below:</para>
+
+          <screen>workflow:
+  steps:
+    - set_flags:
+        flags:
+          - type: build
+            status: enable
+            project: home:Admin
+          - type: publish
+            status: disable
+            project: home:Admin</screen>
+
+          <para>The package, repository and architecture keys are all optional. When provided, they limit the flag to a certain package, repository or architecture.
+          The architecture key depends on the repository key. Other optional keys can be provided by themselves.</para>
+
+          <para>So with the configuration file provided below and a <emphasis>pull request event</emphasis>, builds of the
```

Comment:
```suggestion
          <para>So with the configuration file provided below and a <emphasis>pull request event</emphasis>, the builds of the
```

Reply:
`the` is optional here, so if you agree, I would leave the sentence as it is. What do you think?

Reply:
I'm fine